### PR TITLE
dev-tex/curve: EAPI7 revbump

### PR DESCRIPTION
**Pull request by mm1ke**
Hi,

This is another simple EAPI7 revbump, however i didn't try to compile-test it (yet).
Please review.

Closes: https://bugs.gentoo.org/664256

### CODE REVIEW

File: dev-tex/curve/curve-1.16-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit latex-package
+
+S=${WORKDIR}/${PN}
```
Comment (Zlogene) [inline dev-tex/curve/curve-1.16-r1.ebuild]: move it below

File: dev-tex/curve/curve-1.16-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+DESCRIPTION="LaTeX style for a CV (curriculum vitae) with flavour option"
+HOMEPAGE="https://ctan.org/tex-archive/macros/latex/contrib/curve/"
+SRC_URI="http://mirrors.ctan.org/macros/latex/contrib/${PN}.zip -> ${P}.zip"
+
+LICENSE="LPPL-1.2"
+RDEPEND=">=dev-texlive/texlive-latexextra-2010"
+DEPEND="${RDEPEND}"
+BDEPEND="app-arch/unzip"
```
Comment (Zlogene) [inline dev-tex/curve/curve-1.16-r1.ebuild]: it is not `BDEPEND`

File: dev-tex/curve/curve-1.16-r1.ebuild [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+IUSE="doc examples"
+
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~sparc ~x86"
+
+TEXMF=/usr/share/texmf-site
+DOCS=( README NEWS THANKS )
+
+src_install() {
+	einstalldocs
```
Comment (Zlogene) [inline dev-tex/curve/curve-1.16-r1.ebuild]: dodoc

**Comment by mm1ke** [comment]
ping, any chance we can merge this :)

**Pull request closed without merging**